extlinks/links/models.py

```diff
@@ -77,64 +167,3 @@ class Meta:
     total = models.PositiveIntegerField()
 
 
-class LinkEvent(models.Model):
-    """
-    Stores data from the page-links-change EventStream
-
-    https://stream.wikimedia.org/?doc#!/Streams/get_v2_stream_page_links_change
-    """
-
-    class Meta:
-        app_label = "links"
-        get_latest_by = "timestamp"
-        indexes = [
-            models.Index(fields=["hash_link_event_id",]),
-            models.Index(fields=["timestamp",]),
-        ]
-
-    url = models.ManyToManyField(URLPattern, related_name="linkevent")
```

Comment:
We actually need to keep this field in the model for it to be available for data migration. We'll need to remove it in a followup patch.